tests/bluetooth/tester/testcase.yaml

```diff
@@ -14,8 +14,66 @@ tests:
     platform_allow:
       - rd_rw612_bga
       - frdm_rw612
+      - frdm_mcxw71
     extra_configs:
       - CONFIG_NXP_MONOLITHIC_BT=n
+  bluetooth.general.tester_gatt_long_dev_name:
+    build_only: true
+    harness: bluetooth
+    platform_allow:
+      - frdm_mcxw71
+    extra_configs:
+      - CONFIG_BT_CONN_DISABLE_SECURITY=y
```

Comment:
you can use 
`
extra_configs:
    platform:frdm_mcxw71:CONF_FILE=nxp_bt.conf
`
and move those configs to nxp_bt.conf. and then the platform_allow can be removed